app/controllers/tasks_controller.rb

```diff
@@ -94,9 +94,9 @@ def enforce_policy
   def notify_task_updated!
     if @task_completion_change
       action = task.completed? ? 'complete' : 'incomplete'
-
+      feed_name = task.is_metadata? ? 'manuscript' : 'workflow'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.